### PR TITLE
Renv advice

### PR DESCRIPTION
**Pull request by mratford**

### CODE REVIEW

File: source/documentation/appendix-docs/botor.md [hunk 1]
```diff
@@ -80,6 +80,20 @@ reticulate::py_install('boto3')
...
+If this process goes wrong run
+
+```r
+renv::deactivate()
+```
+
+in the console and 
+
+```bash
+rm -rf renv renv.lock .Rprofile requirements.txt
```
Comment (Th368MoJ) [inline source/documentation/appendix-docs/botor.md:92]: Why `requirements.txt` here? I don't believe we have any steps that create it

Comment (mratford) [reply source/documentation/appendix-docs/botor.md:92]: `renv::snaphot()` will create it if python is used.

Comment (Th368MoJ) [reply source/documentation/appendix-docs/botor.md:92]: Ah, I did not know that

**Review by Th368MoJ — APPROVED** [review]
looks good to me

**Pull request merged**